python/test/test_algo_bench.py

```
@@ -0,0 +1,122 @@
+import argparse
+import contextlib
+import json
+import os
+import sys
+import time
+from collections import namedtuple
+from datetime import datetime
+
+import numpy as np
+import pytest
+import pytz
+from pyarrow import Schema
+
+import katana.benchmarking.bench_python_cpp_algos as katbench
+
+
+def generate_args(
+    json_output,
+    input_dir="./",
+    graph="GAP-road",
+    app="bfs",
+    source_nodes="",
+    trails=1,
+    num_sources=4,
+    thread_spin=False,
+    threads=1,
```

Since this is used in testing, defaults will hide bugs, so don't provide them.